code/src/WijmoProvider/Grid/FlexGrid.ts

```diff
@@ -20,6 +21,33 @@ namespace WijmoProvider.Grid {
             );
         }
 
+        /**
+         * This action performs a workaround for an issue related with
+         * Safari 14.* version. The paint doesn't get triggered by the
+         * scroll on the grid.
+         *
+         * @private
+         * @memberof FlexGrid
+         */
+        private _safari14workaround(): void {
+            if (
+                /^((?!chrome|android).).*Version\/14.*safari/i.test(
+                    navigator.userAgent
+                )
+            ) {
+                this._provider.updatedView.addHandler(
+                    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                    (grid: wijmo.grid.FlexGrid, e: wijmo.EventArgs) => {
+                        //removes previous tranform
+                        grid._root.style.transform = '';
+                        //this is the "fake" transform that forces Safari to repaint the grid area
+                        grid._root.style.transform = 'translateZ(0)';
+                    }
+                );
+                console.log('The fix for Safari 14 has been applied.');
```

Comment:
Should we keep this console.log?

Reply:
I would say yes. I would like to have a clear log, when the fix is applied.